pkg/entity/project/project.go

```diff
@@ -56,3 +61,32 @@ func create(projectName *string, cmd *cobra.Command) *projects.CreateProjectResp
 	common.CliExit(err)
 	return response
 }
+
+func manage(cmd *cobra.Command) {
+	flags := cmd.Flags()
+	membersToAdd, err := flags.GetStringArray(addMemberFlag)
+	membersToRemove, err := flags.GetStringArray(removeMemberFlag)
+	activeProject := GetActiveProject()
+
+	addReq := &projects.AddProjectMembersRequest{
+		Emails:    membersToAdd,
+		ProjectId: activeProject,
+	}
+	removeReq := &projects.RemoveProjectMembersRequest{
+		Emails:    membersToRemove,
+		ProjectId: activeProject,
+	}
+
+	_, err = client.AddProjectMembers(apiContext, addReq)
+	common.CliExit(err)
+	_, err = client.RemoveProjectMembers(apiContext, removeReq)
+	common.CliExit(err)
+	return
+}
+
+func GetActiveProject() string {
```

Comment:
You could use `GetActiveProject` from the context package, I think?

Reply:
It didn't import. That's why I copy-pasted it